debian/packetfence.postinst

```diff
@@ -147,16 +158,15 @@ case "$1" in
         systemctl disable packetfence-iptables
         systemctl enable packetfence-haproxy-admin
         systemctl enable packetfence-tracking-config.path
-        systemctl restart docker
-        systemctl restart packetfence-config
+        systemctl start packetfence-config
         /usr/local/pf/bin/pfcmd generatemariadbconfig --force
+        # only packetfence-config is running after this command
         systemctl isolate packetfence-base.target
 
         # Override the MySQL socket location to match our standard
         printf '[client-server]\nsocket = /var/lib/mysql/mysql.sock\n' > /etc/mysql/mariadb.conf.d/999-socket-override.cnf
         sed -i 's#^socket\s*=.*#socket=/var/lib/mysql/mysql.sock#' /etc/mysql/my.cnf
 
-        perl /usr/local/pf/addons/upgrade/add-default-params-to-auth.pl
```

Comment:
AFAIK, this is necessary to keep, otherwise when we add new parameters to the auth sources, the config entries created on older version become out of sync and this creates various issues

Reply:
@jrouzierinverse, can you confirm what @julsemaan said ?

For me, if we add new parameters to auth sources:
- new sources will be created with new parameters
- old sources will need to be updated using an upgrade script to add this new parameter

Reply:
@jrouzierinverse, any comment ?

Reply:
Script is necessary. I will adjust code.